charts/shopware/templates/store.yaml

```diff
@@ -29,20 +29,39 @@ spec:
   {{- end }}
 
   sessionCache:
-    {{- if .Values.redissession.enabled }}
+    {{- if .Values.valkeysession.enabled }}
```

Comment:
These two conditions are the same minus the value of the host. Would it be simpler if we use the condition and use a common `redisHost` variable that is set by a helper to either Valkey or Redis?

It seems `Redis` or `Valkey` use the "same" variable, so we could just use the as well and point them to either `Redis` or `Valkey`.